eng/common/templates/job/build-from-source.yml

```diff
@@ -0,0 +1,75 @@
+parameters:
+  jobNamePrefix: 'sourcebuild'    # Override for prefix for job names
+  managedOnly: false              # Specifies whether source is managed only. If so, one build platform is sufficient
+  portableOnly: false             # If true, builds one leg with portable flag enabled
```

Comment:
Isn't the behavior the inverse? This disables all legs except the one with the portable flag enabled? (And the one with the portable flag should always run, either way, right?)